Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/__pycache__\r\n/*.pyc\r\n/*.pyo\r\n/venv\r\n/.idea\r\n/db.sqlite3\r\n/.venv/\r\n.idea/
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-16LE
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision ed38c781150d5bf38bea91919901c224f8dce221)
+++ b/.gitignore	(date 1742153732029)
@@ -5,4 +5,10 @@
 /.idea
 /db.sqlite3
 /.venv/
-.idea/
\ No newline at end of file
+.idea/
+/.idea/.gitignore
+/.idea/misc.xml
+/.idea/modules.xml
+/.idea/MystDoku.iml
+/.idea/inspectionProfiles/profiles_settings.xml
+/.idea/vcs.xml
